website/docs/getting-started/tutorial.md

```diff
@@ -92,25 +86,48 @@ In case of `mime-types` it will find the `package.json` file and write the resul
 On the first attempt of running the analyzer on the `mime-types` package it will fail with an error message:
 
 ```shell
-The following package managers are activated:
-        Bower, Bundler, Cargo, Composer, DotNet, Gradle, Maven, NPM, NuGet, PIP, SBT, Stack, Yarn
+The following 25 package manager(s) are enabled:
+        Bazel, Bower, Bundler, Cargo, Carthage, CocoaPods, Composer, Conan, GoMod, Gradle, Maven, NPM, NuGet, PIP, Pipenv, PNPM, Poetry, Pub, SBT, SpdxDocumentFile, Stack, SwiftPM, Unmanaged, Yarn, Yarn2
+The following 2 package curation provider(s) are enabled:
+        DefaultDir, DefaultFile
 Analyzing project path:
-        [mime-types-dir]
-ERROR - Resolving dependencies for 'package.json' failed with: No lockfile found in '[mime-types-dir]'. This potentially results in unstable versions of dependencies. To support this, enable the 'allowDynamicVersions' option in 'config.yml'.
-Writing analyzer result to '[analyzer-output-dir]/analyzer-result.yml'.
+        [workdir]/mime-types
+00:31:07.985 [main] WARN  org.ossreviewtoolkit.plugins.packagemanagers.node.utils.NpmDetection - Any of [NPM, PNPM, YARN, YARN2] could be the package manager for '[workdir]/mime-types/package.json'. Assuming it is an NPM project.
```

Comment:
> Why 4x the same warning?

Because all of NPM, PNPM, YARN, YARN2 individually state via common `NpmDetection` code that none of them is clearly responsible for that definition file.

> Should this be warning just be left out of the tutorial, or addressed by some other means?

*If* quoting console output, I believe it should be complete to avoid confusion about differences. Maybe the only exception should be lengthy / unimportant output that could be replaced by ellipsis (`[...]`).